nbformat/v4/nbformat.v4.schema.json

```diff
@@ -76,6 +76,27 @@
                       },
                       "additionalProperties": true
                     }
+                },
+                "environment": {
```

Comment:
Perhaps making it an ordered list of environment objects (as currently) specified... would give us the ability to store a number of options for different platforms/environment providers, etc...